tools/ThermoRawFileParser/test-data/really_small.indexed_mzML

```diff
@@ -26,8 +26,8 @@
       </referenceableParamGroup>
     </referenceableParamGroupList>
     <softwareList count="1">
-      <software id="ThermoRawFileParser" version="1.3.3">
-        <cvParam cvRef="MS" accession="MS:1003145" name="ThermoRawFileParser" />
+      <software id="ThermoRawFileParser" version="1.3.4">
+        <cvParam cvRef="MS" accession="MS:1000799" value="ThermoRawFileParser" name="custom unreleased software tool" />
```

Comment:
Hi @martenson the offsets are file positions. Since this line got 40 characters longer all offsets are +40

Reply:
Maybe you can ask the developers if they can avoid such changes: 

```
- name="ThermoRawFileParser"
+ value="ThermoRawFileParser" name="custom unreleased software tool"
```

Reply:
That is actually what broke all the downstream parsers. They are rolling it back.

Reply:
details: https://github.com/compomics/ThermoRawFileParser/issues/112#issuecomment-821295866

Reply:
This is part of the fix, we should merge.